src/component/framework/form-group.js

```diff
@@ -31,7 +37,7 @@ export class FormGroup {
    */
   @computedFrom('element')
```

Comment:
It does work. Computedfrom avoids recalculating every cycle by telling that the getter should only be recalculated when a specific bindable changes it's value. `element` is a bindable in the form-group component.

I don't think computedFrom supports the dot notation.